core/services/ocr2/plugins/threshold/decryption_queue.go

```
@@ -206,11 +206,21 @@ func (dq *decryptionQueue) SetResult(ciphertextId decryptionPlugin.CiphertextId,
 
 	req, ok := dq.pendingRequests[string(ciphertextId)]
 	if ok {
-		dq.lggr.Debugf("responding with result for pending decryption request ciphertextId %s", string(ciphertextId))
-		req.chPlaintext <- plaintext
+		if err != nil {
+			dq.lggr.Debugf("decryption error for ciphertextId %s", string(ciphertextId))
```

Is there a reason for suppressing the error message?